types/src/main/scala/roc/types/decoders.scala

```diff
@@ -23,7 +26,7 @@ object decoders {
 
   implicit val stringElementDecoder: ElementDecoder[String] = new ElementDecoder[String] {
     def textDecoder(text: String): String         = text
-    def binaryDecoder(bytes: Array[Byte]): String = bytes.map(_.toChar).mkString
```

Comment:
Yeah, I missed a decent portion of these. I read the documentation, then went to look at the actual "authorized" Postgres JDBC driver, and the only portion of the code I saw was for dealing w/ the standard Short / Int / Long / Float / Double Big Endian format. If I recall ( from somewhere ), I put some of the other "binary" decoders out as the documentation specified that if no Binary version was found, it was assumed to be the Text Version.